### PR TITLE
PAINTROID-258 Refactor tools.common package to Kotlin

### PR DESCRIPTION
**Pull request by CodeChamp-SS**
Refactored tools.common package to Kotlin.

Files that were refactored:
1. CommonBrushChangedListener
2. CommonBrushPreviewListener
3. Constants
4. PointScrollBehavior
5. ScrollBehavior

https://jira.catrob.at/browse/PAINTROID-258

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Paintroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [ ] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [x] Post a message in the *#paintroid* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: Paintroid/src/main/java/org/catrobat/paintroid/tools/common/Constants.kt [hunk 1]
```diff
@@ -16,14 +16,15 @@
...
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+package org.catrobat.paintroid.tools.common
 
-package org.catrobat.paintroid.tools.common;
+class Constants {
+    companion object {
+        const val SCROLL_TOLERANCE_PERCENTAGE = 0.1f
+        const val MOVE_TOLERANCE = 5f
+    }
```
Comment (gPathpp) [inline Paintroid/src/main/java/org/catrobat/paintroid/tools/common/Constants.kt:25]:
```suggestion
const val SCROLL_TOLERANCE_PERCENTAGE = 0.1f
const val MOVE_TOLERANCE = 5f
```
Please omit the class and just write the constants.
https://kotlinlang.org/docs/properties.html#compile-time-constants

File: Paintroid/src/main/java/org/catrobat/paintroid/tools/common/ScrollBehavior.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.paintroid.tools.common
+
+import android.graphics.Point
+
+interface ScrollBehavior {
+    fun getScrollDirection(pointX: Float, pointY: Float, viewWidth: Int, viewHeight: Int): Point?
```
Comment (gPathpp) [inline Paintroid/src/main/java/org/catrobat/paintroid/tools/common/ScrollBehavior.kt:24]:
```suggestion
    fun getScrollDirection(pointX: Float, pointY: Float, viewWidth: Int, viewHeight: Int): Point
```

File: Paintroid/src/main/java/org/catrobat/paintroid/tools/common/CommonBrushPreviewListener.kt [hunk 3]
```diff
@@ -16,41 +16,20 @@
...
+package org.catrobat.paintroid.tools.common
 
-package org.catrobat.paintroid.tools.common;
+import android.graphics.Paint.Cap
+import org.catrobat.paintroid.tools.ToolPaint
+import org.catrobat.paintroid.tools.ToolType
+import org.catrobat.paintroid.tools.options.BrushToolOptionsView.OnBrushPreviewListener
 
-import android.graphics.Paint;
+class CommonBrushPreviewListener(private val toolPaint: ToolPaint, private val toolType: ToolType) :
```
Comment (gPathpp) [inline Paintroid/src/main/java/org/catrobat/paintroid/tools/common/CommonBrushPreviewListener.kt:26]: Well done 👍

**Review by gPathpp — CHANGES_REQUESTED** [review]
Well done 👍 
Please consider these small code improvements. 
Thanks!

File: Paintroid/src/main/java/org/catrobat/paintroid/tools/common/ScrollBehavior.kt (hunk 2 above)
Comment (CodeChamp-SS) [reply Paintroid/src/main/java/org/catrobat/paintroid/tools/common/ScrollBehavior.kt]: done

File: Paintroid/src/main/java/org/catrobat/paintroid/tools/common/Constants.kt (hunk 1 above)
Comment (CodeChamp-SS) [reply Paintroid/src/main/java/org/catrobat/paintroid/tools/common/Constants.kt]: done :)

**Review by gPathpp — APPROVED** [review]
LGTM 👍

**Comment by wslany** [comment]
Thanks!

**Pull request merged**